src/types/line.js

```diff
@@ -190,9 +201,44 @@ export default class LineAnnotation extends Element {
 }
 
 LineAnnotation.id = 'lineAnnotation';
+
+const arrowHeadsDefaults = {
+  backgroundColor: undefined,
+  backgroundShadowColor: undefined,
+  borderColor: undefined,
+  borderDash: undefined,
+  borderDashOffset: undefined,
+  borderShadowColor: undefined,
+  borderWidth: undefined,
+  enabled: undefined,
+  fill: undefined,
+  length: undefined,
+  shadowBlur: undefined,
+  shadowOffsetX: undefined,
+  shadowOffsetY: undefined,
+  width: undefined
+};
+
 LineAnnotation.defaults = {
   adjustScaleRange: true,
-  backgroundShadowColor: 'transparent',
+  arrowHeads: {
```

Comment:
I don't think you replied to this "remove everything else" suggestion:

```suggestion
  arrowHeads: {
    end: Object.assign({}, arrowHeadsDefaults),
    length: 12,
    start: Object.assign({}, arrowHeadsDefaults),
    width: 6
  }
```

does it not work?